DevExtreme CLI: add React Next.js info (#7295)
DevExpress/devextreme-documentation: concepts/50 React Components/50 Application Template/45 Authentication/10 Vite Authentication/10 Implement Authentication Functions.md

> Authentication functions for back-end requests are located in `src\api\auth.js`. You must update these functions to make requests to your server, such as to add or verify users.

Reviewer: The last sentence has a complex structure. It's not immediately clear what noun is related to the "such as" clause. I'd rather have a separate sentence there. 

```suggestion
Authentication functions for back-end requests are located in `src\api\auth.js`. You must update these functions to make requests to your server. For example, server requests can add users or verify their credentials. 
```

> Authentication functions for back-end requests are located in `src\api\auth.js`. You must update these functions to make requests to your server. For example, server requests can add users or verify their credentials. 

Reviewer 2: ```suggestion
Authentication functions for backend requests are in `src\api\auth.js`. You must update these functions to make requests to your server. For example, add users or verify their credentials. 
```

    </tr>
    <tr>
        <td>isOk</td>

>         <td>A Boolean value that represents whether the request was successful.</td>

Reviewer: ```suggestion
        <td>A Boolean value that indicates whether the request was successful.</td>
```
